projects/minizinc/minizinc_fuzzer.dict

```diff
@@ -0,0 +1,50 @@
+reserved_keyword_0="ann"
```

Comment:
tip: you don't actually need to use the X="Y" syntax with dictionaries.
"Y" is sufficient.